src/tests/functional/shared_test_classes/include/shared_test_classes/base/low_precision_transformations/layer_transformation.hpp

```diff
@@ -30,16 +30,16 @@ class LayerTransformationParamsNGraphFactory {
 class LayerTransformationParamsFactory : public LayerTransformationParamsNGraphFactory {
 };
 
-class LayerTransformation : virtual public LayerTestsUtils::LayerTestsCommon {
+class LayerTransformation : virtual public ov::test::SubgraphBaseTest {
+public:
+    static std::pair<double, double> getQuantizationInterval(const ngraph::element::Type precision);
+
 protected:
     LayerTransformation();
 
-    static InferenceEngine::Blob::Ptr GenerateInput(
-        const ngraph::element::Type precision,
-        const InferenceEngine::TensorDesc& tensorDesc,
-        const float k = 1.f);
+    ov::test::utils::InputsMap get_input_map() override;
 
-    static std::pair<float, float> getQuantizationInterval(const ngraph::element::Type precision);
+    ov::test::utils::CompareMap get_compare_map() override;
```

Comment:
why we cannot use the default way?

Reply:
The previous version of `default way` is used. It's not my implementation.
We need additional time to use new comparation logic. Anyway, It's not part of migration to OpenVINO API 2.0.